client/src/app/_components/groups/group-post-item/index.tsx

```diff
@@ -51,31 +110,24 @@ const GroupPostItem = (currentPost: FirebasePostModel) => {
 			}
 		} catch (err) {
 			console.log(err);
+			toast.error('Error fetching likes for post');
```

Comment:
I would maybe use this as toast.warn() because it's not breaking functionality but still needs to be shown to the user